src/scipp/core/reduction.py

```diff
@@ -25,28 +25,39 @@ def mean(x: VariableLike,
 
     See :py:func:`scipp.sum` on how rounding errors for float32 inputs are handled.
 
-    :param x: Input data.
-    :param dim: Dimension along which to calculate the mean. If not
-                given, the mean over all dimensions is calculated.
-    :param out: Optional output buffer.
-    :raises: If the dimension does not exist, or the dtype cannot be summed,
```

Comment:
Missing `raises`?

Reply:
Is it required here? We always raise in these two cases.